### PR TITLE
AIX debug external supported in jdk-options

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Signed-off-by: andrew-m-leonard <andrew_m_leonard@uk.ibm.com>

### CODE REVIEW

File: make/autoconf/jdk-options.m4 [hunk 1]
```diff
@@ -302,8 +302,24 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_DEBUG_SYMBOLS],
       [AS_HELP_STRING([--with-native-debug-symbols],
       [set the native debug symbol configuration (none, internal, external, zipped) @<:@varying@:>@])],
       [
+        if test "x$OPENJDK_TARGET_OS" = xaix; then
+          if test "x$withval" = xexternal || test "x$withval" = xzipped; then
+            # We support external for AIX now
+          fi
+        fi
```
Comment (keithc-ca) [inline make/autoconf/jdk-options.m4:309]: ~This does nothing at all and can be removed.~ It's worse:
```
syntax error near unexpected token `fi'
```

File: make/autoconf/jdk-options.m4 [hunk 2]
```diff
@@ -302,8 +302,24 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_DEBUG_SYMBOLS],
...
       ],
       [
+        if test "x$OPENJDK_TARGET_OS" = xaix; then
+          # AIX doesn't support 'external' so use 'internal' as default
+          with_native_debug_symbols="internal"
```
Comment (keithc-ca) [inline make/autoconf/jdk-options.m4:314]: We want the default to be `external` even on AIX.

File: make/autoconf/jdk-options.m4 (hunk 1 above)
Comment (andrew-m-leonard) [reply make/autoconf/jdk-options.m4:309]: :-)

**Comment by keithc-ca** [comment]
Is this still needed, or can it be closed?

**Comment by andrew-m-leonard** [comment]
not needed

**Pull request closed without merging**